Jenkinsfile

```diff
@@ -125,18 +121,24 @@ def CloudInstallAndTest(cloudTarget) {
   def nodeReq = "ubuntu && amd64 && ${cloudTarget}"
   node(nodeReq) {
     echo "Installing DLR package for ${cloudTarget} target"
-    withAWS(credentials:'Neo-AI-CI-Fleet') {
-      files = s3FindFiles(bucket: 'neo-ai-dlr-jenkins-artifacts', glob: "${env.JOB_NAME}/${env.BUILD_ID}/artifacts/dlr-*-manylinux1_x86_64.whl")
-      assert files.size() == 1
-      files.each {
-        s3Download file: it.name, bucket: 'neo-ai-dlr-jenkins-artifacts', path: it.path, force: true
-      }
+    if (cloudTarget == "p2" || cloudTarget == "p3") {
+      unstash 'dlr_gpu_whl'
+    } else {
+      unstash 'dlr_cpu_whl'
     }
     sh """
-    ls -lh *.whl
-    pip3 install *.whl
-    pip3 install tensorflow
+    ls -lh python/dist/*.whl
+    pip3 install python/dist/*.whl
     """
+    if (cloudTarget == "p2" || cloudTarget == "p3") {
+      sh """
+      pip3 install tensorflow_gpu
+      """
+    } else {
+      sh """
+      pip3 install tensorflow
```

Comment:
Just merged PR #55 where I changed the command to
```sudo pip3 install --upgrade --force-reinstall tensorflow```

Reply:
I have just merged the latest commit